### ISSUE TITLE
Parse empty tags as self-closing tags (part 2)

### ISSUE THREAD

**Issue by lcnvdl** [NaturalIntelligence/fast-xml-parser#277]
Hi! I'm doing this issue based on the last one.

Maybe I made a bad explanation. Please, let me try again.

I'm trying to doing this:

input:
```javascript
const jObj = {
            a: {
                b: [
                    "",
                    {c: "val2"}
                ]
            }
        };
        const parser = new Parser({
            supressEmptyNode: false
        });
        const result = parser.parse(jObj);
        console.log(result);
```

I'm getting the next output:
```html
<a><b></b><b><c>val2</c></b></a>
```

But I really need:
```html
<a><b /><b><c>val2</c></b></a>
```

Please, note the "b" with the self closing tag. I'm trying to get that output.

Is this possible with this library?

I've tried to get that output, but I can't find an option to do this.

**Comment by amitguptagwl** [comment]
I hope I'm able to understand you this time

```js
const jObj = {
            a: {
                b: [
                    "",
                    {c: "val2"}
                ]
            }
        };
        const parser = new Parser({
            supressEmptyNode: true
        });
        const result = parser.parse(jObj);
        console.log(result);
```
Output
```xml
<a><b/><b><c>val2</c></b></a>
```

Feel free to correct me.

**Comment by amitguptagwl** [comment]
closing as no update. feel free to reopen it if issue exist.
